src/pages/Home/index.tsx

```diff
@@ -13,15 +13,18 @@ const HomePageComponent = () => (
   <div className="content">
     <div className="element-wrapper" />
     <div className="passage">
-      <h1 className="passage-title">Welcome to Content Spec Explorer!</h1>
+      <h1 className="passage-title">Welcome!</h1>
       <div className="passage-content">
-        The CSE website provides easy access to a host of Smarter Balanced’s test development
-        documents. The primary resources in the CSE are the Item Specifications (“Item Specs”). Item
-        Specs outline how assessment items are to be written so the response shows evidence that the
-        students have the skills/knowledge to perform the item/task. The Item Specs are used by the
-        test development team to produce consistent, well-aligned items across the content areas.
-        Educators are encouraged to implement a similar structure when preparing classroom-level
-        assessments for both summative and formative uses.
+        The Content Spec Explorer is a tool for educators to learn about the design and content of
```

Comment:
Is there a reason for this line not having p tag unlike the other two lines below?